css/src/admin-global.css

```diff
@@ -838,6 +838,9 @@ body.folded .wpseo-admin-submit-fixed {
   color: #d63638;
 }
 
+.privacy-settings #yoast-premium-deactivated-notice {
```

Comment:
We might consider having this rule for all notices rather than just the premium deactivated notice to make sure that we don't run into this problem again if we choose to ever add another notification there.

Reply:
Done 👍